docs/api_docs/python.rst

```diff
@@ -0,0 +1,170 @@
+
+Installing with Pipenv
+-----------------------------
+In order to install the Vertiq python API into your pipenv virtual environment, simply run ``pipenv install iqmotion`` within your virtual environment.
+
+Installing with Pip
+-----------------------------
+In order to install the Vertiq python API via pip, simply run ``pip install iqmotion``.
+
+Python API Basics
+=========================
+Hardware Setup
+----------------
+Information about required hardware for API communication can be found in the :ref:`Control Center documentation <connection_guide>`. The requirements for API communication 
+are the same as those for the IQ Control Center.
+
+Opening a Serial Connection
+--------------------------------
+In order to connect with your USB-to-UART bridge, the Python API provides the ``SerialCommunicator`` class.
+
+The ``SerialCommunicator`` class is initialized with a serial port name and a baud rate. The name of your serial port is dependent on your operating system. 
+For example, in the hardware example provided in the Control Center documentation linked above, the serial port name is “COM3,” and in Linux, it may be "/dev/ttyUSB0."
+
+In order to create a new ``SerialCommunicator`` object on a Windows operating system where your module is set to 115200 baud, and your FTDI reports on COM9, you would do the following:
+
+.. code-block:: python
+    
+    import iqmotion as iq
+    com = iq.SerialCommunicator("COM9", baudrate=115200)
+
+In order to communicate with a Vertiq module through the Python API, you must create a ``SerialCommunicator`` object.
+
+Creating a Module Object
+-----------------------------
+Vertiq's module objects are preconfigured to communicate with all accessible IQUART clients for your module's firmware. All module objects are created with the same 
+initialization.
+
+1. A serial communicator object
+2. Optionally, a module ID (``module_idn``)
+
+  * This represents the module ID of the device you would like to communicate with. If your module has its module ID configured to 26, you must set ``module_idn`` to 26 in object instantiation. 
+    By default, all modules have an ID of 0.
+
+3. Optionally, a firmware style (``firmware``)
+
+  * This value is a string, and can be “speed”, “servo”, or “pulsing”. To learn more about what firmware styles are available for your module, please refer to your module's family page.
+
+4. Optionally, a path to additional clients (``clients_path``)
+
+  * There are two options available for adding extra clients. First, in order to add all client files in a folder, you can pass a path to a folder that contains all additional, client json files. Second, 
+    you can pass an array of paths to custom client json files in order to only add those you are interested in rather than an entire folder.
```

Comment:
It may be worth mentioning in a note or something that using additional clients is pretty uncommon for customers. I just don't want people to think they should be typically using additional client files.